lib/utils/proxy-utils.ts

```diff
@@ -0,0 +1,36 @@
+import * as nutil from 'node:util/types';
+
+export type OneArgFn<T> = (arg: any) => T;
+
+export const isDynamicProxy = Symbol("isDynamicProxy");
+export const sourceFunction = Symbol("sourceFunction");
+
+
```

Comment:
Could we have documentation to these functions pls?

Reply:
addressed